CHANGELOG

```diff
@@ -1,5 +1,375 @@
+* Fix #10472: Add myself back as codeowners  (#10576)
+* Separate lighthouse accessibility checks into a new suite (#10571)
+* Remove unnecessary ids and update names for ids used for accessibility. (#10570)
+* Fixes Lint error in lighthouse_setup script (#10568)
+* Fix #10556: Added Math SVGs for demo explorations. (#10564)
+* Add link to code change guidelines in PR template (#10559)
+* Fixing position of the question help box on mobile. (#10554)
+* Fix part of #8668: document logger and messenger services (#10544)
+* Fix part of #9632: Introduce/fix jobs for regenerating the missing stats of an Exploration (#10541)
+* Fix #10522: Add initialization at page startup for PlatformFeatureService (#10505)
+* Fix part of #9827: Update the way backend python libraries are installed. (#10502)
+* Fix part of #10474: Cover more files with strict checks (#10485)
+* Adds a lint check to restrict the usage of bypassing angular's security mechanism, decode the URI encoded profile pictures and remove safe-pipe.ts (#10484)
+* Fix part of #10474, Fix #7434: Cover more files with strict checks and Makes all constants accessible in AppConstants using dot notation (#10476)
+* Milestone 3.4: Achieve Lighthouse scores of 100 in topic and skill dashboard, topic editor, topic_viewer, and thanks page (#10461)
+* Rewrite the coreEditorAndPlayerFeatures Suite (#10443)
+* Fix backend test error introduced in #10021 (#10401)
+* Fix #10045 : Remove the temporary math SVG generation functions  (#10393)
+* Fix #10359: Removes waitForNavigation() from lighthouse puppeteer scripts (#10382)
+* Add test cases for PlatformParameter registry (#10368)
+* Fix #9379: Added validation for svg_filename field in math rich-text components. (#10366)
+* Fixes backend test flake in SuggestionSvgFilenameValidationOneOffJobTests (#10330)
+* Fix #10072: Adds dtslint tests to test that customization argument backend interfaces match the customization argument interfaces (#10326)
+* Fix #10324: Fixing space in navbar (#10325)
+* Removed inline styling (#10287)
+* Replace self for Android codeowners while on vacation (#10275)
+* Fixes flaky frontend tests (#10270)
+* Fix #10247: Set CodeMirror viewportMargin to 100 and fix e2e flake (#10248)
+* Fixes docstr in customization_args_util_test.py (#10244)
+* Fix #10235: Fixing Broken Frontend Tests Wiki Link (#10237)
+* Fix #10094: Only create exploration opportunity on topic/story publish. (#10212)
+
+Translations
+* Fix #10098: Translation tab fixes (#10870)
+* Reverses the AnswerGroup domain object migration to allow translations of rule inputs from PR10246 (#10516)
+* Adds frontend components to contribute interaction customization args translations in the translation tab of exploration editor (#10308)
+* Migrates AnswerGroup domain object to allow translations of rule inputs (#10246)
+
+Server Errors
+* Suppress AppEnginePlatformWarning warning (#10871)
+* Removes unwanted suggestion cron jobs  (#10423)
+
+
```

Comment:
Could you also please add, here:

```
v3.0.3 (CANCELLED)
-----------------------------
This release was cancelled due to the release preparation process running significantly past the deadline.
```

This will help other devs who look at this in the future and wonder where 3.0.3 is. (Please add blank two lines before and after this section too, for parity with what's already there.)

Thanks!

Reply:
Done.